### PR TITLE
fixed image hover in left corner and client x-y

### PR DESCRIPTION
**Pull request by Agies**

### CODE REVIEW

**Comment by timruffles** [comment]
Thanks - will take a look!

File: ios-drag-drop.js [hunk 1]
```diff
@@ -136,8 +136,10 @@
       var touch = event.changedTouches[0];
       var x = touch[coordinateSystemForElementFromPoint + 'X'];
       var y = touch[coordinateSystemForElementFromPoint + 'Y'];
-      dropEvt.offsetX = x - target.x;
-      dropEvt.offsetY = y - target.y;
+      dropEvt.clientX = x;
+	  dropEvt.clientY = y;
+	  dropEvt.offsetX = x - event.target.clientLeft;
+	  dropEvt.offsetY = y - event.target.clientTop;
```
Comment (xStrom) [inline ios-drag-drop.js]: Why is this ```event.target``` more correct than just ```target``` which is passed to the function?

Comment (Agies) [inline ios-drag-drop.js]: It probably is not, what really matters its the clientx and clienty, as they are part of the drop event spec from what I read. I believe offset could be removed in favor of screen x and y.

https://developer.mozilla.org/en-US/docs/Web/Events/drop

Comment (xStrom) [inline ios-drag-drop.js]: If ```event.target.clientLeft``` here can be replaced with ```target.clientLeft``` then I think we should do that.

I haven't looked too closely at how (if at all) the targets differ here, but I do see that we spend some considerable effort deriving ```target``` via ```elementFromTouchEvent```.

Comment (Agies) [inline ios-drag-drop.js]: I tested the switch to use target.clientXXX everything works out the same, so I have switched the code.

**Comment by xStrom** [comment]
The left corner fix here is no longer needed or even valid, as it was fixed with #36. So the following code should be removed from this patch:
```javascript
this.dragImage.style["left"] = "-999";
this.dragImage.style["top"] = "-999";
```

**Comment by timruffles** [comment]
Thanks again for the contribution! If you could take a look at the comment we'd be happy to merge.

**Pull request closed without merging**